src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search search-string-big"
+      data-qa="big"
+    >
+      <label class="search__label search__label-big">
+        <input
+          class="search__input search__input-big"
```

Comment:
```suggestion
          class="search__input"
```

Reply:
no need to add --big modifier.

set styles for `big` input in `search__input` selector
then rewrite in search__input-small some properties